dashboard/src/views/Projects/components/TableRow/index.tsx

```diff
@@ -22,7 +21,7 @@ export const ProjectTableRow: FC<ProjectTableRowProps> = ({
   const createdAtDate = formatIsoDateToLocaleString(createdAt);
 
   return (
-    <tr>
+    <tr onClick={() => window.location.href = `/projects/${projectId}/versions`}>
```

Comment:
I'm pretty certain that won't be accessible, you have to to make it semantically behave as an `a` tag. Look at [this](https://stackoverflow.com/questions/64776155/make-table-row-or-cell-be-read-by-a-table-as-clickable) stackoverflow post

Reply:
This is the internal project. In this case, if someone needs to click the row - the button is accessible through the keyboard